scripts/conversion_tools/unit_conversion.py

```diff
@@ -113,7 +113,35 @@ def kg_kg_minus_1__to__g_kg_minus_1():
 
 def g_kg_minus_1__to__kg_kg_minus_1():
     """Convert gram per kilogram to kilogram per kilogram"""
-    return '{var}/1.0E+3{kind}'
+    return '1.0E-3{kind}*{var}'
+
+##################
+# Plane angle    #
+##################
+
+def radian__to__degrees():
+    """Convert radian to degrees"""
+    return '57.295779513{kind}*{var}'
+
+def degrees__to__radian():
+    """Convert degrees to radian"""
+    return '{var}/57.295779513{kind}'
```

Comment:
While correct, this will eventually run afoul of efforts to standardize constants.
We should revisit (e.g., via `pi / 180.0{kind}` and its inverse) this then (i.e., when a universal value for pi is available to CCPP modules).

Reply:
Yes, I agree. I was wondering if I should plug in the value of math.pi and then "calculate" 57.x as 180.0/pi, but that seemed to be unnecessary to me. Rather implement it correctly once pi is defined and available as a Fortran constant through the CCPP framework. Then one can set pi = 1 and the world suddently looks a lot more familiar ;-)

Reply:
We should definitely add that to the physical constants dictionary. What good would it be if there was no version with c == &#x210F; == 1?
c == &#x210F; == &#960; == 1 is only for advanced users.